datafusion/sql/src/parser.rs

```diff
@@ -704,6 +706,10 @@ impl<'a> DFParser<'a> {
             self.parser
                 .parse_keywords(&[Keyword::IF, Keyword::NOT, Keyword::EXISTS]);
         let table_name = self.parser.parse_object_name(true)?;
+        let temporary = self
```

Comment:
I am not sure if we need the temporary flag for an external table, as [sqlparser-rs](https://github.com/sqlparser-rs/sqlparser-rs/blob/51cbd5a3e69f9cc903f91d7df335658094b5e811/src/parser/mod.rs#L3576) does not support it.
If we want to support it, the correct syntax is best as `create temp external table et ...`.

Reply:
would it make sense to create an issue in `sqlparser-rs` for this? @jonahgao ?

Reply:
ah yes i missed that, thank you